_layouts/default.html

```diff
@@ -8,6 +8,9 @@
 </head>
 <body>
     <header>
+        <a href="{{ '/' | relative_url }}" style="text-decoration: none; margin-right: 10px;">
+            <img src="{{ '/assets/images/home-icon.png' | relative_url }}" alt="Home" style="width: 24px; height: 24px; vertical-align: middle;">
```

Comment:
Instead of an img here, use this SVG, appropriately sized for this line:
`<svg fill="#000000" width="800px" height="800px" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M21.555,8.168l-9-6A1,1,0,0,0,12,2h0a1,1,0,0,0-.554.168l-9,6A1,1,0,0,0,3,10H4V21a1,1,0,0,0,1,1H19a1,1,0,0,0,1-1V10h1a1,1,0,0,0,.556-1.832ZM18,20H6V8.2l6-4,6,4Z"/></svg>`